singlestoredb/tests/test_connection.py

```diff
@@ -1357,6 +1357,18 @@ def test_results_format(self):
                     out = cur.fetchall()
                     assert type(out[0]) is dict, type(out)
 
+    def test_multi_statements(self):
```

Comment:
I see the HTTP tests failed. I'm guessing multi-statements are not possible in the Data API, so this test may have to just be skipped when that protocol is being used.

Reply:
yeah, now I've added a check for that